tests/dualtor_mgmt/test_egress_drop_nvidia.py

```diff
@@ -0,0 +1,247 @@
+    back to the lower tor and then sent to the server B through the active port.
+
+    And the traffic on the lower tor should be dropped by the egress drop acl rule because the server A port
+    on the lower tor is standby.
+    """
+    with allure.step("Prepare test parameters"):
+        dualtor_meta = dualtor_info(ptfhost, upper_tor_host, lower_tor_host, tbinfo)
+        lower_tor_mg_facts = lower_tor_host.get_extended_minigraph_facts(tbinfo)
+        vlan_interface = list(lower_tor_mg_facts['minigraph_vlans'].keys())[0]
+        # use the last port as the active port on lower tor
+        lower_tor_active_port = lower_tor_mg_facts['minigraph_vlans'][vlan_interface]['members'][-1]
+        lower_tor_active_server = lower_tor_mg_facts['minigraph_neighbors'][lower_tor_active_port]['name']
+        lower_tor_active_server_ip = lower_tor_mg_facts['minigraph_devices'][lower_tor_active_server]['lo_addr']
+        lower_tor_active_server_ptf_port = lower_tor_mg_facts['minigraph_ptf_indices'][lower_tor_active_port]
+
+        lower_tor_t1_ptf_ports = []
+        lower_tor_vlan_ptf_ports = []
+        for portchannel in lower_tor_mg_facts['minigraph_portchannels'].values():
+            for member in portchannel['members']:
+                ptf_indice = lower_tor_mg_facts['minigraph_ptf_indices'][member]
+                lower_tor_t1_ptf_ports.append(ptf_indice)
+        # use the first standby server on lower tor
+        lower_tor_vlan_ports = lower_tor_mg_facts['minigraph_vlans'][vlan_interface]['members']
+        for vlan_port in lower_tor_vlan_ports:
+            lower_tor_vlan_ptf_ports.append(lower_tor_mg_facts['minigraph_ptf_indices'][vlan_port])
+        lower_tor_standby_port = lower_tor_mg_facts['minigraph_vlans'][vlan_interface]['members'][0]
+        lower_tor_standby_server_ptf_port = lower_tor_mg_facts['minigraph_ptf_indices'][lower_tor_standby_port]
+
+    with allure.step("Switch the last server on the lower tor to active"):
+        toggle_simulator_port_to_lower_tor(lower_tor_active_port)
+        wait_until(10, 5, 0, check_muxcable_status, upper_tor_host, lower_tor_active_port, "standby")
+        wait_until(10, 5, 0, check_muxcable_status, lower_tor_host, lower_tor_active_port, "active")
+
+    with allure.step("Clear port counters"):
+        upper_tor_host.command("sonic-clear counters")
+        lower_tor_host.command("sonic-clear counters")
+
+    with allure.step("Start the traffic test, "
+                     "check the traffic received by the last server is the bounced back traffic from the upper tor"):
+        pkt = simple_tcp_packet(
+            eth_dst=dualtor_meta['vlan_mac'],
+            ip_src='1.1.1.1',
+            ip_dst=lower_tor_active_server_ip,
+            ip_ttl=64
+        )
+        # Send packets
+        ptfadapter.dataplane.flush()
+        testutils.send(ptfadapter, lower_tor_standby_server_ptf_port, pkt, PKT_NUM)
```

Comment:
`lower_tor_standby_server_ptf_port` seems to be one of the vlan ports on lower tor, and the mux status should be `standby`. I think the packet should be dropped by the egress rules. Why we still expect the packet being forwarded?

Reply:
This test case covers the scenario that server A is active on the upper tor and is standby on the lower tor, server B is standby on the upper tor and active on the lower tor, and the upstream traffic is sent from server A to server B.

Here the lower_tor_standby_server is the server A, which is also the "upper_tor_active_server". And the ptf port number for the server is shared by both tors, so when we use the testutils to send the traffic, it will be duplicated to both tors by the mux-simulator. And the traffic will be dropped by the lower tor(server A is standby), but will be forwarded by the upper tor through the tunnel(server A is active, the destination server B is standby), and then fowarded by the lower tor to the server B(active on lower tor).

So I expect that the bounced back traffic should be received by the lower_tor_active_server_ptf_port, and there is no direct forwarded traffic from server A to server B on the lower tor(This is checked by the ttl of the received packets, [Line100-line108](https://github.com/sonic-net/sonic-mgmt/pull/9619/files/b8b5b2152e6d8028eaf9c9ed01bbace15ad4d2e8#diff-24f8b7ba2d894c20f16ef5408f9264cc7cec8b1b43c756ec2b65d993c0356ec7R100-R108) )

And at last, I check the port counters to double confirm there is no duplicated traffic received by the server B.
![image](https://github.com/sonic-net/sonic-mgmt/assets/97947969/bfb0b9f5-daa5-4785-8142-be85eb9fcab3)

Reply:
Thanks! It's very clear now.